docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+---
+comments: true
+---
+
+# Scaling Workloads with KEDA based on Keptn metrics
+
+If you want to use [KEDA](https://keda.sh/) to scale your workloads, you can set it up
+to consume metrics from Keptn.
+This gives you great flexibility for your setup since Keptn can
+consolidate different observability providers for you and KEDA
+will simplify the scaling operations for you.
+
+This use case is enabled by the
+[Custom Metrics API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
```

Comment:
Since for  Keda  we are directly using the metrics endpoint of Keptn (e.g. http://metrics-operator-service.keptn-system.svc.cluster.local:9999/api/v1/metrics/podtato-kubectl/cpu-throttling), i think we can leave out the mentioning of the Kubernetes API extension, to avoid confusion

Reply:
hmm good point, i'll try to rewrite it a bit

Reply:
done